lib/main_common.pm

```diff
@@ -1426,6 +1426,9 @@ sub load_extra_tests_desktop {
         }
         if (!get_var("NOAUTOLOGIN")) {
```

Comment:
btw, I don't understand why to schedule the next modules only under this condition. `!NOAUTOLOGIN` means only on `AUTOLOGIN`. Especially your new test module seems to switch it off anyway.

Reply:
I just scheduled it next to the `multi_users_dm` test because this module is similar.

Reply:
Also AFAIK we don't run `extra_tests_on_gnome` without autologin anyway. As this test module assumes this, it makes sence to run it only in this case as it would fail otherwise.